Index: python-nacl-1.3.0/tests/test_kx.py
===================================================================
--- python-nacl-1.3.0.orig/tests/test_kx.py
+++ python-nacl-1.3.0/tests/test_kx.py
@@ -16,6 +16,7 @@ from __future__ import absolute_import,
 
 from hypothesis import given, settings
 from hypothesis.strategies import binary
+from hypothesis import HealthCheck
 
 import pytest
 
@@ -48,7 +49,7 @@ def test_crypto_kx_seed_keypair(seed1, s
 @given(binary(min_size=33,
               max_size=128),
        )
-@settings(max_examples=20)
+@settings(suppress_health_check=[HealthCheck.too_slow],max_examples=20)
 def test_crypto_kx_seed_keypair_seed_too_large(seed):
     with pytest.raises(exc.TypeError):
         b.crypto_kx_seed_keypair(seed)
